### PR TITLE
More cleanup based on MISRA style guide.

### PR DESCRIPTION
**Pull request by nomis52**
https://en.wikipedia.org/wiki/Motor_Industry_Software_Reliability_Association

### CODE REVIEW

File: firmware/src/network_model.c [hunk 1]
```diff
@@ -260,8 +260,9 @@ int NetworkModel_GetDHCPMode(const RDMHeader *header,
   unsigned int offset = sizeof(RDMHeader);
   memcpy(g_rdm_buffer + offset, param_data, INTERFACE_ID_SIZE);
   offset += INTERFACE_ID_SIZE;
-  g_rdm_buffer[offset++] =
+  g_rdm_buffer[offset] =
```
Comment (peternewman) [inline firmware/src/network_model.c]: This seems a lot clearer.

File: firmware/src/rdm_handler.c [hunk 2]
```diff
@@ -31,7 +31,7 @@
 #include "utils.h"
 #include "system_pipeline.h"
 
-#define MAX_RDM_MODELS 6
+enum { MAX_RDM_MODELS = 4};
```
Comment (peternewman) [inline firmware/src/rdm_handler.c]: Not 6 still?

File: firmware/src/transceiver.c [hunk 3]
```diff
@@ -231,17 +231,6 @@ static inline void RebaseTimer(uint16_t last_event) {
                            PLIB_TMR_Counter16BitGet(TMR_ID_3) - last_event);
 }
 
-/*
```
Comment (peternewman) [inline firmware/src/transceiver.c]: No longer required?

File: firmware/src/transceiver.c [hunk 4]
```diff
@@ -322,8 +312,9 @@ void UART_FlushRX() {
 bool UART_RXBytes() {
   while (PLIB_USART_ReceiverDataIsAvailable(g_hw_settings.usart) &&
          g_transceiver.data_index != BUFFER_SIZE) {
-    g_transceiver.active->data[g_transceiver.data_index++] =
+    g_transceiver.active->data[g_transceiver.data_index] =
       PLIB_USART_ReceiverByteReceive(g_hw_settings.usart);
```
Comment (peternewman) [inline firmware/src/transceiver.c]: Indent four from g_tran

File: firmware/src/transceiver.c (hunk 3 above)
Comment (peternewman) [inline firmware/src/transceiver.c]: Do they not like inlines?

**Comment by peternewman** [comment]
I wonder if we can lint this <digit>u bit somehow?

File: firmware/src/network_model.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/network_model.c]: Yup

File: firmware/src/rdm_handler.c (hunk 2 above)
Comment (nomis52) [inline firmware/src/rdm_handler.c]: I've bumped it up since I'm added more models.

File: firmware/src/transceiver.c (hunk 3 above)
Comment (nomis52) [inline firmware/src/transceiver.c]: It was more of a case that this was only used from one location.

File: firmware/src/transceiver.c (hunk 4 above)
Comment (nomis52) [inline firmware/src/transceiver.c]: Done

**Comment by nomis52** [comment]
I've asked coverity if we can enable MISRA checking on this repo. I think they only offer as it a commercial tool but it's worth a shot.

File: firmware/src/rdm_handler.c (hunk 2 above)
Comment (peternewman) [inline firmware/src/rdm_handler.c]: That's what I'm saying, this commit pushes it back down to 4.

File: firmware/src/transceiver.c (hunk 3 above)
Comment (peternewman) [inline firmware/src/transceiver.c]: Understood.

File: firmware/src/rdm_handler.c (hunk 2 above)
Comment (nomis52) [inline firmware/src/rdm_handler.c]: Oh, good point. I'll increase it in the next change again.

**Comment by peternewman** [comment]
Nice, that would be cool if they did it.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**